bodhi/server/services/updates.py

```diff
@@ -536,6 +538,23 @@ def new_update(request):
 
             if len(releases) > 1:
                 result = dict(updates=updates)
+
+            if from_tag:
```

Comment:
Wondering if we should restrict that to the release that are not composed by bodhi (ie rawhide). I don't think we want to allow create update from side tag for all releases yet.
@pypingou  what do you think ?

Reply:
>  I don't think we want to allow create update from side tag for all releases yet.

This sounds wise, let's start with rawhide only, it should be fairly straight forward to change this later on. This would allow using chain-build in stable releases which is a side-effect of gating rawhide packages that we are interested in, but I'd rather we do this in small steps :)

Reply:
@Zlopez then to do that you will need to check that the update release is not composed by bodhi. You can check here how it was done --> https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/scripts/approve_testing.py#L116

Reply:
Ok, I will update it